students/johnpharmd/lesson09/mailroom_OO.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+import sys
+
+
+class Donor:
+    """creates objects for individual donors"""
+    def __init__(self, title, last_name, donation):
+        # fixme: self.ID = 0
+        self.title = title
+        self.last_name = last_name
+        self.donations = [donation]
+        super().__init__()
+
+    @property
+    def donor(self):
+        """getter for individual donor"""
+        return {self.last_name: {'title': self.title, 'donations':
+                sum(self.donations), 'num_donations': len(self.donations)}}
+
+    @property
+    def title(self):
+        return self.new_title
+
+    @title.setter
+    def title(self, new_title):
+        """enables title change for donor"""
+        self.new_title = new_title
```

Comment:
if you're not doing anything special with an attribute then you don't need a property (it is also discouraged to defined self attributes outside of `__init__` method like `self.new_title`)
you can access title directly.